### ISSUE TITLE
Editor js freezes when trying to render toggle block

### ISSUE THREAD

**Issue by megazoll** [kommitters/editorjs-toggle-block#155]
### Precheck

- [x] Do a quick search and make sure the bug has not been reported yet.
- [x] Please disclose security vulnerabilities privately at [oss@kommit.co](mailto:oss@kommit.co).
- [x] Finally, be nice and have fun!.

### Description

When adding toggle block to editor, saving data and then trying to render data back, editor js breaks down. Looks like an infinite loop somewhere in plugin.

### Environment

* Editor.js version: 2.25.0
* Plugins you use with their versions: toggle-block
* Device, Browser, OS: any (Chrome 105)

### Current behavior

https://codepen.io/megazoll/pen/VwxzMQx

### Expected behavior

Editorjs should render toggle block

**Comment by untilhamza** [comment]
### Cause of this issue
* Inside the `renderItems()` method there seems to be an infinite while loop in the `else` case of the `if`  statement 

```
 if (this.readOnly) {
      const redactor = document.getElementsByClassName("codex-editor__redactor")[0];

      const { children } = redactor;

      const { length } = children;

      for (let i = 0; i < length; i += 1) {

        const blockCover = children[i].firstChild;
        const blockContainer = blockCover.firstChild;
        const { id } = blockContainer;

        if (id === this.wrapper.id) {
          toggleRoot = i;
          break;
        }
      }
    } else {
      const toggle = this.wrapper.children[1];
      let currentBlock = {};

      while (currentBlock[1] !== toggle) {

        //***** INFO: this is an infinite loop *****

        toggleRoot = this.api.blocks.getCurrentBlockIndex();
        const block = this.api.blocks.getBlockByIndex(toggleRoot);
        const { holder } = block;
        const blockCover = holder.firstChild;
        const blockContent = blockCover.firstChild;
        currentBlock = blockContent.children;

        this.api.caret.setToNextBlock("end", 0);
      }
    }
```

**Comment by untilhamza** [comment]
### Cause of this issue
* Inside the `renderItems()` method there seems to be an infinite while loop in the `else` case of the `if`  statement 

```
 if (this.readOnly) {
      const redactor = document.getElementsByClassName('codex-editor__redactor')[0];
      const { children } = redactor;
      const { length } = children;

      for (let i = 0; i < length; i += 1) {
        const blockCover = children[i].firstChild;
        const blockContainer = blockCover.firstChild;
        const { id } = blockContainer;

        if (id === this.wrapper.id) {
          toggleRoot = i;
          break;
        }
      }
    } else {
      const toggle = this.wrapper.children[1];
      let currentBlock = {};

      while (currentBlock[1] !== toggle) {
        // 🚨🚨***** INFO: this is an infinite loop ***** 🚨🚨
        toggleRoot = this.api.blocks.getCurrentBlockIndex();
        const block = this.api.blocks.getBlockByIndex(toggleRoot);
        const { holder } = block;
        const blockCover = holder.firstChild;
        const blockContent = blockCover.firstChild;
        currentBlock = blockContent.children;

        this.api.caret.setToNextBlock('end', 0);
      }
    }
```

**Comment by yimin087** [comment]
I also met

**Comment by miguelnietoa** [comment]
Thank you all for reporting the bug, we will prioritize it!

**Comment by miguelnietoa** [comment]
Hi there! This bug has been solved in the `v0.3.1` 😅

**Comment by untilhamza** [comment]
> Hi there! This bug has been solved in the `v0.3.1` 😅 

Thank you so much 😇

**Comment by yimin087** [comment]
Thank you so much
